releases/2023.2/2023.2.md

```diff
@@ -0,0 +1,182 @@
+
+
+## Kytos-ng and NApps released
+
+The following Kytos-ng projects and NApps were released in this version. For further detailed information about what was released and what changed, please visit the release page of each project and look for the `CHANGELOG.rst` file.
+
+
+Project                                                             | Release Web Page
+------------------------------------------------------------------- |------------------------------------------------------------------------
+[kytos](https://github.com/kytos-ng/kytos)                          | [kytos](https://github.com/kytos-ng/kytos/releases)
+[python-openflow](https://github.com/kytos-ng/python-openflow)      | [python-openflow](https://github.com/kytos-ng/python-openflow/releases)
+[kytos-utils](https://github.com/kytos-ng/kytos-utils)              | [kytos-utils](https://github.com/kytos-ng/kytos-utils/releases)
+[ui](https://github.com/kytos-ng/ui)                                | [ui](https://github.com/kytos-ng/ui/releases)
+[kytos/of_core](https://github.com/kytos-ng/of_core)                | [kytos/of_core](https://github.com/kytos-ng/of_core/releases)
+[kytos/flow_manager](https://github.com/kytos-ng/flow_manager)      | [kytos/flow_manager](https://github.com/kytos-ng/flow_manager/releases)
+[kytos/topology](https://github.com/kytos-ng/topology)              | [kytos/topology](https://github.com/kytos-ng/topology/releases)
+[kytos/of_lldp](https://github.com/kytos-ng/of_lldp)                | [kytos/of_lldp](https://github.com/kytos-ng/of_lldp/releases)
+[kytos/pathfinder](https://github.com/kytos-ng/pathfinder)          | [kytos/pathfinder](https://github.com/kytos-ng/pathfinder/releases)
+[amlight/coloring](https://github.com/kytos-ng/coloring)            | [kytos-ng/coloring](https://github.com/kytos-ng/coloring/releases)
+[amlight/sdntrace_cp](https://github.com/kytos-ng/sdntrace_cp)      | [kytos-ng/sdntrace_cp](https://github.com/kytos-ng/sdntrace_cp/releases)
+[kytos/mef_eline](https://github.com/kytos-ng/mef_eline)            | [kytos/mef_eline](https://github.com/kytos-ng/mef_eline/releases)
+[kytos/of_multi_table](https://github.com/kytos-ng/of_multi_table)  | [kytos/of_multi_table](https://github.com/kytos-ng/of_multi_table/releases)
+[kytos/telemetry_int](https://github.com/kytos-ng/telemetry_int)    | [kytos/telemetry_int](https://github.com/kytos-ng/telemetry_int/releases)
+
+### Switches Compatibility List
+
+The following switches have been tested with Kytos version `2023.2`:
+
+Switch Vendor                        | Switch          | Firmware version        | OpenFlow version | Environment type
+-------------------------------------|-----------------|-------------------------|------------------|-----------------
+[OVS](https://www.openvswitch.org/)  | ovs-switchd     | 2.15                    | 1.3              | testing
+[NoviFlow](https://www.noviflow.com) | WB5132-F        | NW570.2.1 and NW550.2.3 | 1.3              | production
```

Comment:
This is also assuming that the same NoviFlow firmware version is being used

Reply:
For now I'll leave NoviFlow switchs out from this table, it'll be added again later on https://github.com/kytos-ng/kytos-ng.github.io/issues/71